pkg/apis/forklift/v1beta1/register.go

```diff
@@ -33,3 +36,19 @@ var SchemeGroupVersion = schema.GroupVersion{
 }
 
 var SchemeBuilder = &scheme.Builder{GroupVersion: SchemeGroupVersion}
+
+// TODO: find a better place for this, it would be nice to use it also to
+// determine the plural configuration of a resource for the operator
+func GetGroupResource(required runtime.Object) (group string, resource string, err error) {
+
+	switch required.(type) {
+	case *Provider:
+		group = SchemeGroupVersion.Group
+		resource = "providers"
```

Comment:
This might not be related to this patch directly but...
What should the users be able to do in a forklift?
This checks if they can list/get the providers, what about the mappings?
Should all the users be able also to create all those CRs?

Reply:
it depends on the permissions the users have, users with create/update permissions on providers, plans, migrations, mappings should be able to do everything an admin can do with forklift :)